docker-compose.yml

```diff
@@ -4,25 +4,29 @@ services:
       dockerfile: vtn.Dockerfile
       context: .
     ports:
-      - "127.0.0.1:3000:3000"
+      - ${VTN_PORT}:3000
```

Comment:
Is there a reason you want to bind to any interface instead of just loopback? It does not necessarily seem smart to me to expose the development environment to an external interface.
Same for the DB in line 27

Reply:
The issue with loopback is that depending on the host system it may be harder to run. For instance if someone is building on WSL they will probably run into issues with loopback. While slightly less secure this does ensure that developers can more quickly pick up and run the project regardless how the host is configured.

Reply:
The other consideration is that binding in this manner allows multiple instances to run at once without blocking each other. This is useful if you want to iterate on a hot reloaded server with cargo watch while running integration tests against a separate docker compose. But that may be a premature optimization at this stage of the project.

Reply:
I was not really aware of the issue with WSL, as I don't use that myself. I think it's a valid argument to keep it the way you proposed, as there are no significant security implications anyway.